syntax = "proto3";

package smartfarm;


option go_package = "github.com/fermat01/smart-farm-iot-platform/pb;pb";

// Import sensor.proto to reuse Ack message
import "sensor.proto";

service ActuatorService {
  rpc SetPump(PumpCommand) returns (Ack) {}
}

message PumpCommand {
  string action = 1;   // "ON" or "OFF"
  string sensor_id = 2;
}


